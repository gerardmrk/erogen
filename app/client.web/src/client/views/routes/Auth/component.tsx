import * as React from "react";
import styles from "./component.styles.scss";
import HeadTags from "@client/views/components/HeadTags";
import Container from "@client/views/components/ui-elements/Container";
import { LocalProps } from ".";

export type Props = LocalProps;

export type State = {};

export class Auth extends React.PureComponent<Props, State> {
  public static i18nNamespace = "route_Auth";

  public render() {
    return (
      <React.Fragment>
        <HeadTags
          path={this.props.path}
          title={this.props.t("title")}
          description={this.props.t("description")}
          metaType={this.props.metaType}
          metaImgPath={this.props.metaImgPath}
          metaImgAlt={this.props.metaImgAlt}
          metaTwitterCardType={this.props.metaTwitterCardType}
        />

        <div className={styles.main}>
          <Container>
            <h1>{"Auth"}</h1>
          </Container>
        </div>
      </React.Fragment>
    );
  }
}
