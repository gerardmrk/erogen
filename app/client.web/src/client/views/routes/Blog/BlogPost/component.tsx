import * as React from "react";
import styles from "./component.styles.scss";

type Props = {};

type State = {};

export class BlogPost extends React.PureComponent<Props, State> {
  public static readonly chunkName = "Blog-BlogPost";
  public static readonly i18nNamespace = "";

  public render() {
    return (
      <div className={styles.main}>
        <h1>{"BlogPost"}</h1>
      </div>
    );
  }
}

export default BlogPost;
