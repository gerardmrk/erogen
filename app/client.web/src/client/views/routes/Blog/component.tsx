import * as React from "react";
import styles from "./component.styles.scss";
import { Switch } from "react-router-dom";
import { LocalProps } from ".";
import EnhancedRoute from "@client/views/components/EnhancedRoute";
import Container from "@client/views/components/ui.elements/Container";
import HeadTags from "@client/views/components/HeadTags";

export type Props = LocalProps;

export type State = {};

export class Blog extends React.PureComponent<Props, State> {
  public static i18nNamespace = "route_Blog";

  public render() {
    return (
      <React.Fragment>
        <HeadTags
          path={this.props.path}
          title={this.props.t("title")}
          description={this.props.t("description")}
          metaType={this.props.metaType}
          metaImgPath={this.props.metaImgPath}
          metaImgAlt={this.props.metaImgAlt}
          metaTwitterCardType={this.props.metaTwitterCardType}
        />

        <div className={styles.main}>
          <Container>
            <h1>{"Blog"}</h1>

            <Switch>
              {this.props.routes.map((r, i) => (
                <EnhancedRoute key={i} {...r} />
              ))}
            </Switch>
          </Container>
        </div>
      </React.Fragment>
    );
  }
}
