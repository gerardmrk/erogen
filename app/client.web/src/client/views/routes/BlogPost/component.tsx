import * as React from "react";
import styles from "./component.styles.scss";
import { LocalProps } from ".";
import HeadTags from "@client/views/components/HeadTags";

type Props = LocalProps;

type State = {};

export class BlogPost extends React.PureComponent<Props, State> {
  public static i18nNamespace = "route_BlogPost";

  public render() {
    return (
      <React.Fragment>
        <HeadTags
          path={this.props.path}
          title={this.props.t("title")}
          description={this.props.t("description")}
          metaType={this.props.metaType}
          metaImgPath={this.props.metaImgPath}
          metaImgAlt={this.props.metaImgAlt}
          metaTwitterCardType={this.props.metaTwitterCardType}
        />

        <div className={styles.main}>
          <h1>{"BlogPost"}</h1>
        </div>
      </React.Fragment>
    );
  }
}
