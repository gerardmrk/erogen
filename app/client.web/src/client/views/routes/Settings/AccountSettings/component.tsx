import * as React from "react";
import styles from "./component.styles.scss";

type Props = {};

type State = {};

export class AccountSettings extends React.PureComponent<Props, State> {
  public render() {
    return (
      <div className={styles.main}>
        <h1>{"AccountSettings"}</h1>
      </div>
    );
  }
}

export default AccountSettings;
