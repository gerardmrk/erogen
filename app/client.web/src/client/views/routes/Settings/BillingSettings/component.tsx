import * as React from "react";
import styles from "./component.styles.scss";

type Props = {};

type State = {};

export class BillingSettings extends React.PureComponent<Props, State> {
  public static readonly chunkName = "";
  public static readonly i18nNamespace = "";

  public render() {
    return (
      <div className={styles.main}>
        <h1>{"BillingSettings"}</h1>
      </div>
    );
  }
}

export default BillingSettings;
