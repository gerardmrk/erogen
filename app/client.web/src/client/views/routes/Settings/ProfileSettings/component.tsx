import * as React from "react";
import styles from "./component.styles.scss";

type Props = {};

type State = {};

export class ProfileSettings extends React.PureComponent<Props, State> {
  public static readonly chunkName = "";
  public static readonly i18nNamespace = "";

  public render() {
    return (
      <div className={styles.main}>
        <h1>{"ProfileSettings"}</h1>
      </div>
    );
  }
}

export default ProfileSettings;
