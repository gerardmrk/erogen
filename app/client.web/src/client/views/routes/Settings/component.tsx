import * as React from "react";
import styles from "./component.styles.scss";
import { LocalProps } from ".";

type Props = LocalProps;

type State = {};

export class Settings extends React.PureComponent<Props, State> {
  public render() {
    return (
      <div className={styles.main}>
        <h1>{"Settings"}</h1>
      </div>
    );
  }
}
