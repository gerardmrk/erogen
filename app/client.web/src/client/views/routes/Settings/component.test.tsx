import * as React from "react";
import { create, ReactTestRenderer, ReactTestInstance } from "react-test-renderer"; // prettier-ignore

import { Settings } from "./component";
import { withTranslationMock } from "@client/views/_fixtures_/wrapper";

const Component = withTranslationMock()(Settings);

describe("<Settings/>", () => {
  let renderer: ReactTestRenderer;
  let component: ReactTestInstance;

  beforeEach(() => {
    renderer = create(<Component />);
    component = renderer.getInstance() as ReactTestInstance;
  });

  it("renders ok", () => {
    expect(component).toBeDefined();
  });
});
