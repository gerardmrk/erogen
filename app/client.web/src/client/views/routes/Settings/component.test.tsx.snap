// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Settings/> snapshot 1`] = `
<div
  className="main"
>
  <h1>
    Settings
  </h1>
</div>
`;
